mutt/PKGBUILD

```diff
@@ -4,6 +4,7 @@
 pkgname=mutt
 pkgver=1.9.3
 pkgrel=1
+epoch=1
```

Comment:
Git for Windows does not modify `mutt` Therefore this change should be unnecessary.